File: Latihan3/Latihan3_Java/Ram.java
public class Ram {

	// class attribute
	String capacity;
	int price;

	// construcotrs
	Ram() {
	}

	Ram(String capacity, int price) {
		this.capacity = capacity;
		this.price = price;
	}

	// getter and setter
	public void setCapacity(String capacity) {
		this.capacity = capacity;
	}

	public String getCapacity() {
		return capacity;
	}

	public void setPrice(int price) {
		this.price = price;
	}

	public int getPrice() {
		return price;
	}

}

File: Latihan3/Latihan3_Python/Ram.py
class Ram:
    # class attribute
    __capacity = ""
    __price = 0

    # constructors
    def __init__(self, capacity="", price=0):
        self.__capacity = capacity
        self.__price = price

    # getter and setter
    def setCapacity(self, capacity):
        self.__capacity = capacity

    def getCapacity(self):
        return self.__capacity

    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price
